docs/changelog.rst

```diff
@@ -8,10 +8,59 @@ This section lists features and improvements of note in each release.
 
 The full release history can be viewed `at the GitHub perses releases page <https://github.com/choderalab/perses/releases>`_.
 
+0.9.3 - Release
+---------------
+
+Bugfixes
+^^^^^^^^
+
+- (PR `#894 <https://github.com/choderalab/perses/pull/894>`_)
+  Remove unused argument 'implicitSolvent' from SystemGenerator in tests.
+
+- (PR `#893 <https://github.com/choderalab/perses/pull/893>`_)
+  Add installation instructions to readme.
+
+- (PR `#892 <https://github.com/choderalab/perses/pull/892>`_)
+  Allow `generate_dipeptide_top_pos_sys` to accept `demap_CBs`.
+
+- (PR `#878 <https://github.com/choderalab/perses/pull/878>`_)
+  Fix stochastic failures in RepartitionedHybridTopologyFactory test.
+
+- (PR `#877 <https://github.com/choderalab/perses/pull/877>`_)
+  Fix naked charge padding (sigmas for hydroxyl hydrogens are changed from 1.0 nm to 0.06 nm).
+
+- (PR `#874 <https://github.com/choderalab/perses/pull/874>`_)
+  Added readme instructions on how to run perses examples using the docker container with GPUs/CUDA.
+
+- (PR `#866 <https://github.com/choderalab/perses/pull/866>`_)
+  Fix endstate validation handling in PointMutationExecutor.
+
+- (PR `#860 <https://github.com/choderalab/perses/pull/860>`_)
+  Simplify `_construct_atom_map` for protein mutations.
+
+- Various CI fixes
+  * PR `#787 <https://github.com/choderalab/perses/pull/787>`_
+  * PR `#850 <https://github.com/choderalab/perses/pull/850>`_
+  * PR `#858 <https://github.com/choderalab/perses/pull/858>`_
+  * PR `#868 <https://github.com/choderalab/perses/pull/868>`_
+  * PR `#871 <https://github.com/choderalab/perses/pull/871>`_
+  * PR `#880 <https://github.com/choderalab/perses/pull/880>`_
+  * PR `#887 <https://github.com/choderalab/perses/pull/887>`_
+  * PR `#898 <https://github.com/choderalab/perses/pull/898>`_
+
+New features
+^^^^^^^^^^^^
+
+- (PR `#896 <https://github.com/choderalab/perses/pull/896>`_)
+  Drop support for older OpenMM versions.
+  We now only support versions >= 7.6.0.
```

Comment:
Here we need to talk about the new benchmark CLI tools #924 .

I guess it could go something like

```
Command line utility to automatically run and analyze benchmarks using the data set found in https://github.com/openforcefield/protein-ligand-benchmark/ (#924 )
```